pytransform3d/transform_manager/_utils.py

```diff
@@ -0,0 +1,23 @@
+import numpy as np
+
+from .. import transformations as pt
+
+
+def find_neighboring_indices_around_value(arr, value):
```

Comment:
I'm not sure what this function does at the moment. Variable names like arr and value don't help me to understand it.

Reply:
Adapted it to `find_neighboring_timesteps(time_arr, query_time)`